spec/features/create_the_review_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+feature "User want create a post" do
+  let!(:user) { create(:user) }
+  scenario "user on page with all book review and create a book with valid credentials" do
+
+    sign_in(user)
+
+    visit books_path
+
+    expect(page).to have_content "New Book"
```

Comment:
you clicking this in next line, so you dont need to check it. Anyway you'll receive error in case there is no such element